draft-ietf-quic-tls.md

```diff
@@ -924,8 +924,10 @@ based on the client's initial Destination Connection ID, as described in
 The keys used for packet protection are computed from the TLS secrets using the
 KDF provided by TLS.  In TLS 1.3, the HKDF-Expand-Label function described in
 Section 7.1 of {{!TLS13}} is used, using the hash function from the negotiated
-cipher suite.  Other versions of TLS MUST provide a similar function in order to
-be used with QUIC.
+cipher suite.  Note that labels, which are described using strings, are encoded
+as bytes using ASCII {{?ASCII=RFC0020}} without quotes or any trailing NUL
+byte.  Other versions of TLS MUST provide a similar function in order to be
```

Comment:
```suggestion
bytes.  Other versions of TLS MUST provide a similar function in order to be
```

Reply:
This is singular intentionally, to avoid problems like might arise from:

```c
const char* label = "quic key";
hkdf_expand_label(...., label, strlen(label), ...);
```